kore/test/Test/Kore/Parser/Lexeme.hs

```diff
@@ -323,105 +320,6 @@ stringLiteralParserTests =
             ]
         ]
 
-charLiteralParserTests :: [TestTree]
```

Comment:
Where did these go?

Reply:
I just removed them, as we don't have any `CharLiteral` type; however, we have plenty of `StringLiteral` tests so we should be covered